xml/art_solution_automation.xml

```diff
@@ -166,6 +166,175 @@ https://confluence.suse.com/x/8IEnGw
     </variablelist>
   </sect1>
 
+  <sect1 xml:id="sec-sol-autom-hana-using-salt-formulas">
+    <title>Using &salt; Formulas</title>
+    <para>
+      &salt; formulas can be applied in two ways:
+    </para>
+    <itemizedlist>
+      <listitem>
+        <formalpara>
+          <title>&saltmaster; with &salt; Client</title>
+          <para>
+            All steps in <xref linkend="pro-sol-autom-hana-apply-salt-formulas"/> must be
+            executed on the &saltmaster; machine.</para>
+        </formalpara>
+      </listitem>
+      <listitem>
+        <formalpara>
+          <title>&salt; Client only</title>
+          <para>
+            All steps in <xref linkend="pro-sol-autom-hana-apply-salt-formulas"/> must be
+            executed in all of the &salt; clients where the formulas are going to executed.
+          </para>
+        </formalpara>
+      </listitem>
+    </itemizedlist>
+
+    <procedure xml:id="pro-sol-autom-hana-apply-salt-formulas">
+      <title>Applying &salt; Formulas</title>
+      <para>
+        In this procedure we show how an example how to create a pillar structure for
```

Comment:
It looks like something is wrong in this paragraph

Reply:
Argh, yes, you are right. :+1: